status-im/status-react
# [ISSUE #3208] Removed ContextMenu usage

fixes #3208
edit: also #3244 
### Summary:

Moved away from ContextMenu usage

### Steps to test:
- Open Status
- Make general sanity check tests
- Validate all ContextMenu have been migrated to popup lists (e.g. in group chat settings - user profile, when showing profile keys)

status: ready
